corehq/apps/integration/payments/views.py

```diff
@@ -86,6 +88,43 @@ def get_queryset(self):
         query = self._apply_filters(query)
         return query
 
+    def get_context_data(self, **kwargs):
+        context_data = super().get_context_data(**kwargs)
+
+        context_data['user_or_cases_verification_statuses'] = self._get_user_or_cases_verification_status(
+            context_data['page_obj'].object_list
+        )
+
+        return context_data
+
+    def _get_user_or_cases_verification_status(self, object_list):
```

Comment:
What kind of objects are in `object_list`? I can see that it comes from `context_data['page_obj']` but I can't tell what they are. `case_or_user_list`? `kyc_user_list`? Some other kind of object? (If only we had type hints. :smirk: :roll_eyes: )

This is also a very long method name. Nothing other than users and cases can be verified, so I don't think it would be confusing if you left that part out. `_get_verification_statuses(self, foo_list)`?

Reply:
nvm, I see that `object_list` is ES search results.

Reply:
Yes, it could be either depending on how the results are fetched.
In this case however these are from ES

Reply:
Naming updated here, https://github.com/dimagi/commcare-hq/pull/36147/commits/3e200359447620632547fd29df5b5a5549932399, thanks for the suggestion, definitely seems much better this way.